core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -654,6 +662,12 @@ public MetricKey build() {
           .build();
 
   // Metadata sync metrics
+  public static final MetricKey MASTER_METADATA_SYNC_UFS_MOUNT =
+      new Builder("Master.MetadataSyncUfsMount.")
+          .setDescription("The number of UFS operations for a given mount point")
```

Comment:
```suggestion
          .setDescription("The number of UFS sync operations for a given mount point")
```
Do you count both GetStatus and ListChildren?

Reply:
Yes GetStatus and ListChildren.
And also getACL and getFingerprint (which will call get status usually)